docs/src/pages/[platform]/components/selectfield/react.mdx

```diff
@@ -175,6 +177,41 @@ A disabled field will not be focusable or mutable, and it will not be submitted
   </ExampleCode>
 </Example>
 
+### Multiple selections
+
+Use the `isMultiple` prop to specify that multiple options can be selected at once.
+* For Mac: Hold down the command button to select multiple options
+* For Windows: Hold down the control button to select multiple options
+
+<Example>
+  <SelectFieldMultipleStateExample />
+  
+  <ExampleCode>
+
+```jsx file=./examples/SelectFieldMultipleStateExample.tsx
+
+```
+
+  </ExampleCode>
+</Example>
+
+### Select size
+
+The `size` attribute sets the number of visible options in a drop-down list. Use the `selectSize` prop to set the `size` attribute value. If the value of the `selectSize` prop is greater than 1, but lower than the total number of options in the list, the browser will add a scroll bar to indicate that there are more options to view.
```

Comment:
This is a bit confusing to me. Am I supposed to use `size` or `selectSize` to control the number of visible options in a drop-down list? I think what you are trying to say is that the HTML size attribute controls the number of visible options in a drop-down list, but it might be clearer not to talk about that at all and only mention the `selectSize` prop, since that's what users will be using for this.

Reply:
Agree, removed `size` reference, so only `selectSize` is used. May be for a future improvements to avoid this confusion we can swap functionality between those props, so our custom prop size would have a native html attr size functionality.